system-tests/projects/create-react-app-js/src/Foo.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export default function Foo () {
```

Comment:
We don't support codegen for non default exports yet, but I'd like to see support for this soon.